### DISCUSSION TITLE
Not able to see the github stats again

### DISCUSSION THREAD

**Discussion by Rizwans-github** [anuraghazra/github-readme-stats#3288] [Q&A] [answered]
### Describe the bug

It's the same issue that the community is facing since last week.
<img width="1440" alt="Screenshot 2023-09-25 at 10 03 44 AM" src="https://github.com/anuraghazra/github-readme-stats/assets/141806496/4d714c98-e4f8-4581-8064-292385548337">

**Comment by qwerty541** [comment, marked as answer]
Hey, @Rizwans-github! You can see this when upstream GitHub API respond with error due to some reason during fetching stats card data. It should be resolve automatically after some time. Recently we merged https://github.com/anuraghazra/github-readme-stats/pull/3238 which improved confusing behavior of showing 0 commits, currently we show error card instead.
